internal/client/client.go

```diff
@@ -100,30 +105,33 @@ func Run(addr, fingerprint, proxyAddr string) {
 
 	sshPriv, sysinfoError := keys.GetPrivateKey()
 	if sysinfoError != nil {
-		log.Fatal("Getting private key failed: ", sysinfoError)
+		log.Println("Getting private key failed: ", sysinfoError)
 	}
 
 	l := logger.NewLog("client")
 
-	var username string
-	userInfo, sysinfoError := user.Current()
-	if sysinfoError != nil {
-		l.Warning("Couldnt get username: %s", sysinfoError.Error())
-		username = "Unknown"
-	} else {
-		username = userInfo.Username
+	if username == "" {
+		userInfo, sysinfoError := user.Current()
+		if sysinfoError != nil {
+			l.Warning("Couldnt get username: %s", sysinfoError.Error())
+		} else {
+			username = userInfo.Username
+		}
 	}
 
-	hostname, sysinfoError := os.Hostname()
-	if sysinfoError != nil {
-		hostname = "Unknown Hostname"
-		l.Warning("Couldnt get host name: %s", sysinfoError)
-	}
+	/*
```

Comment:
This removes core functionality from RSSH, as it will no longer be able to show what hostname the device has.

Reply:
But this host name is not really important? Because this usage scenario is deployed in an unfamiliar subnet, but we can consider dealing with the authentication user name on the client side

Reply:
That isnt solid reasoning, in doing this you're giving less information to the user who is looking at the client. Plus changes have not been made on the server side to accommodate this.

Reply:
> But this host name is not really important? Because this usage scenario is deployed in an unfamiliar subnet, but we can consider dealing with the authentication user name on the client side

Identifying the hostname of a client is one of the most cruicial parts of many RSSH usecases, including mine.

Please don't remove features from open-source projects just because they don't meet *your* needs, at the expense of everyone else.

Ontop of that, it is inappropriate to add breaking changes like this in a PR, when it is not related to the functionality in the PR.

PRs should only include the functionality discussed in related issues, and anything else should be clearly disclosed in the body of the PR, with justification.

Reply:
Sorry, I've reverted this in a new commit